npm/react/src/mount.ts

```diff
@@ -41,6 +41,36 @@ const injectStyles = (options: MountOptions) => {
  **/
 export const mount = (jsx: React.ReactNode, options: MountOptions = {}) => _mount('mount', jsx, options)
 
+const versionRe = /(\d.+?)\./
```

Comment:
Had the same thought on the semver library - I am more conscious about adding dependencies that bundle each time the user runs a test, since they could impact spec time. We grab their internal regexp: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

I think our regexp do the same thing - I can definitely add some unit tests around this regexp, now I've figured out the hard parts w.r.t the dependencies and system tests.

Reply:
I ended up adding semver since we can just import function-by-function which is nice and small